projects/rero/ng-core/src/lib/service/title-meta.service.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Invenio angular core
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Injectable } from '@angular/core';
+import { Title, Meta } from '@angular/platform-browser';
+import { TranslateService } from '@ngx-translate/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TitleMetaService {
+
+  private _prefix = null;
+
+  /**
+   * Constructor
+   * @param titleService - Title
+   * @param metaService - Meta
+   * @param translateService - TranslateService
+   */
+  constructor(
+    private titleService: Title,
+    private metaService: Meta,
+    private translateService: TranslateService
+  ) { }
+
+  /**
+   * Prefix of title window
+   * @param prefix - string
+   * @return this
+   */
+  setPrefix(prefix?: string) {
+    this._prefix = this.translateService.instant(prefix);
+
+    return this;
+  }
+
+  /**
+   * Title window <head>
+   * @param title - sting
+   * @return this
+   */
+  setTitle(title: string) {
+    let pageTitle = '';
+    if (this._prefix !== null) {
+      pageTitle += this._prefix + ': ';
+    }
+    pageTitle += this.translateService.instant(title);
```

Comment:
As Transifex is now integrated, can you extract the strings ?

Reply:
You must add the translation to the file manual_translation.txt

Reply:
Ok noted